File: Codeforces rounds/Codeforces Beta Round 85 (Div. 2 Only)/A.py
# https://codeforces.com/contest/112/problem/A

def solve(s: str, t: str) -> int:
    s = s.lower()
    t = t.lower()

    if s < t:
        return -1
    elif s > t:
        return 1
    else:
        return 0


def main():
    s: str = input()
    t: str = input()

    ans: int = solve(s, t)
    print(ans)


if __name__ == '__main__':
    main()

File: Codeforces rounds/Codeforces Beta Round 85 (Div. 2 Only)/A.cpp
// https://codeforces.com/contest/112/problem/A

#include <bits/stdc++.h>

using namespace std;

int solve(string a, string b) {
  for (char&c : a) {
    c = tolower(c);
  }
  for (char&c : b) {
    c = tolower(c);
  }

  if (a < b) {
    return -1;
  } else if (a > b) {
    return 1;
  } else {
    return 0;
  }
}

int main(void) {
  ios_base::sync_with_stdio(false);
  cin.tie(NULL);

  string s, t;
  cin >> s >> t;

  int ans = solve(s, t);
  cout << ans << '\n';

  return EXIT_SUCCESS;
}
